File: 0070-climbing-stairs/0070-climbing-stairs.c
int climbStairs(int n) {
    if(n==1){
        return 1;
    }
    if(n==2){
        return 2;
    }
    int a=1,b=2;
    for(int i=3;i<=n;i++){
        int t = a+b;
        a = b;
        b = t;
    }
    return b;
}

File: 0070-climbing-stairs/0070-climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        int [] arr = new int[n+1];

        if(n < 2){
            return n;
        }

        arr[0] =0;
        arr[1]=1;
        arr[2]=2;

        for(int i = 3 ;i <= n ; i++){
            arr[i] = arr[i-1] +arr [i-2];
        }
        return arr[n];
    }
}